tests/unit/src/main/scala/tests/BaseCompletionLspSuite.scala

```diff
@@ -172,13 +177,9 @@ abstract class BaseCompletionLspSuite(name: String) extends BaseLspSuite(name) {
       )
       _ <- assertCompletion(
         """
-          |locally {
```

Comment:
Could you add another `assertCompletion` instead of changing this one?

Reply:
oops, that how wasn't intended

Reply:
fixed